### ISSUE TITLE
[Tarea 2 - Pregunta 2] Uso de random

### ISSUE THREAD

**Issue by diegoheg** [UC-IIC3253/2021#45]
Hola!

Para la implementación de RSA, en específico para los tests de Miller-Rabin está permitido utilizar la librería "random" de python, o es necesario que utilicemos alguna otra librería para generar números de manera pseudo aleatoria?

**Comment by marceloarenassaavedra** [comment]
Sí, está permitido usar la librería random para generar números aleatorios.

Saludos!
